probes/notCreatedRecently/impl.go

```diff
@@ -50,24 +52,20 @@ func Run(raw *checker.RawResults) ([]finding.Finding, string, error) {
 }
 
 func negativeOutcome() ([]finding.Finding, string, error) {
-	f, err := finding.NewWith(fs, Probe,
-		"Repository was created in last 90 days.", nil,
-		finding.OutcomeNegative)
+	text := fmt.Sprintf("Repository was created in last %d days.", lookBackDays)
+	f, err := finding.NewWith(fs, Probe, text, nil, finding.OutcomeNegative)
 	if err != nil {
```

Comment:
Do we need to create a finding `WithValue` even if it's a negative outcome here? That's how it's done with other probes in this PR, and it may be helpful to display it to the user.

Reply:
12199913a8b57303c32072c64434164b9cc5bb2a